addon/components/base/bs-progress.js

```diff
@@ -11,9 +11,9 @@ import layout from 'ember-bootstrap/templates/components/bs-progress';
  Bootstrap, use the appropriate properties like `type`, `showLabel`, `striped` or `animate`.
 
  ```hbs
- {{#bs-progress as |p|}}
-   {{p.bar value=progressValue minValue=0 maxValue=10 showLabel=true type="danger"}}
- {{/bs-progress}}
+  <BsProgress as |P| >
```

Comment:
IMO `P` is a bad name even if uppercase cause of confusion with `<p>` tag. Would recommend to use `progress` or something similar instead.